pkg/contextkey/snapshotter_key.go

```diff
@@ -0,0 +1,24 @@
+package key
```

Comment:
Can we put the package to a more proper place?

Reply:
Could you give some advise